framework/src/modules/chain/transport.js

```diff
@@ -223,80 +223,35 @@ class Transport {
 	attachSharedMethods() {
 		return {
 			/**
-			 * Description of blocksCommon.
-			 *
-			 * @todo Add @param tags
-			 * @todo Add @returns tag
-			 * @todo Add description of the function
+			 * Returns the common blocks between query.ids and the database blocks table
+			 * @param query
+			 * @param query.ids
+			 * @return {Promise<Array<String>>}
 			 */
-			async blocksCommon(query) {
-				query = query || {};
-
+			async getCommonBlocks(query) {
 				const valid = library.schema.validate(
 					query,
-					definitions.WSBlocksCommonRequest
+					definitions.WSGetCommonBlocksRequest
```

Comment:
maybe rename this definition to `GetCommonBlocksRequest`? it's not related to WS anymore